simplic-corelib/Simplic.CoreLib/Text/LevenshteinDistance.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Simplic.Text
+{
+    /// <summary>
+    /// Levensthein algorithm
+    /// </summary>
+    internal static class LevenshteinDistance
+    {
+        /// <summary>
+        /// Compare string

```

Comment:
more information needed here. explain the Levenshtein distance algorythm maybe ? or at least put a link to a wiki page